user_test.go

```diff
@@ -0,0 +1,18 @@
+package discordgo
+
+import "testing"
+
+func TestUser(t *testing.T) {
+	t.Parallel()
+
+	t.Run("String()", func(t *testing.T) {
```

Comment:
1.6 doesn't support Run so Travis is breaking, could you just inline the inside of this method to TestUser? We don't **need** this test to run in parallel :)